### ISSUE TITLE
Django documentation on configuration settings misleading?

### ISSUE THREAD

**Issue by lukik** [celery/celery#4352]
The django documentation seems to be throwing  me off. It is written:

> The uppercase name-space means that all Celery configuration options must be specified in uppercase instead of lowercase, and start with CELERY_, so for example the task_always_eager setting becomes CELERY_TASK_ALWAYS_EAGER, and the broker_url setting becomes CELERY_BROKER_URL.
> 
> You can pass the object directly here, but using a string is better since then the worker doesn’t have to serialize the object.
> 
> app.config_from_object('django.conf:settings', namespace='CELERY')

From [the configuration page](http://docs.celeryproject.org/en/latest/userguide/configuration.html#id8) it shows the changes in 4.1 as

> CELERYBEAT_SCHEDULE  ==> beat_schedule
> CELERYBEAT_SCHEDULER  ==> beat_scheduler
> CELERYBEAT_SCHEDULE_FILENAME  ==> beat_schedule_filename
> 

**Edited:**

My app has the following default setting:

`app.config_from_object('django.conf:settings')`

With the above configuration and given what is currently in the documentation, I expect the following to work (which they aren't)

```
CELERY_TASK_QUEUES = ('myapp.does_something',)
CELERY_BEAT_SCHEDULE = {
    'myapp.does_something.do_something:run-every-10-seconds':
        {'task': 'myapp.does_something.do_something',
         'schedule': timedelta(seconds=10)
}
```

On the other hand, I'd expect the following settings not to work while right now they are the ones that are working

```
CELERY_QUEUES = ('myapp.does_something',)
CELERYBEAT_SCHEDULE = {
    'myapp.does_something.do_something:run-every-10-seconds':
        {'task': 'myapp.does_something.do_something',
         'schedule': timedelta(seconds=10)
}
```
Or what in that explanation am I not understanding?

[Related issue](https://github.com/celery/celery/issues/4209)

Packages in use:
django==1.11.6
celery==4.1.0

**Comment by lukik** [comment]
I think I answered myself. But let me leave it here for anyone else. 

What I wasn't getting was the text [`the code expects new-style names when using namespace= kwarg`](https://github.com/celery/celery/issues/4209#issuecomment-323607805)

So I changed from: 

`app.config_from_object('django.conf:settings')` 

To: `app.config_from_object('django.conf:settings', namespace="CELERY")` 

With that change, settings like `CELERY_TASK_QUEUES` and `CELERY_BEAT_SCHEDULE` now work.
